SPECS/openssl/openssl.spec

```diff
@@ -51,94 +44,18 @@ Patch11:  0011-Remove-EC-curves.patch
 Patch12:  0012-Disable-explicit-ec.patch
 # # Skipped tests from former 0011-Remove-EC-curves.patch
 Patch13:  0013-skipped-tests-EC-curves.patch
-# # Instructions to load legacy provider in openssl.cnf
+# # Instructions to load ly provider in openssl.cnf
 # AZL: NOTE: Had to change this patch because of cascading changes from previous AZL note(s)
 Patch24:  0024-load-legacy-prov.patch
-# # We load FIPS provider and set FIPS properties implicitly
-Patch32:  0032-Force-fips.patch
-# # Embed HMAC into the fips.so
-Patch33:  0033-FIPS-embed-hmac.patch
-# # Comment out fipsinstall command-line utility
-Patch34:  0034.fipsinstall_disable.patch
 # # Skip unavailable algorithms running `openssl speed`
 Patch35:  0035-speed-skip-unavailable-dgst.patch
-# # Extra public/private key checks required by FIPS-140-3
-Patch44:  0044-FIPS-140-3-keychecks.patch
-# # Minimize fips services
-# AZL: NOTE: Removed this because it is RHEL-specific, but want a record of it until I do the final
-#            review of patches.
-#Patch45:  0045-FIPS-services-minimize.patch
-# # Execute KATS before HMAC verification
-Patch47:  0047-FIPS-early-KATS.patch
 # # Selectively disallow SHA1 signatures rhbz#2070977
 # AZL: NOTE: Had to change this patch because of cascading changes from previous AZL note(s)
 Patch49:  0049-Allow-disabling-of-SHA1-signatures.patch
 # # Support SHA1 in TLS in LEGACY crypto-policy (which is SECLEVEL=1)
 Patch52:  0052-Allow-SHA1-in-seclevel-1-if-rh-allow-sha1-signatures.patch
```

Comment:
I am curious if we can remove this patch in a future PR. My thinking:
1. We shouldn't allow SHA1 generally
2. We don't supply a crypto-policy setting like Fedora or RHEL

Reply:
we almost certainly can; I'm targeting FIPS specifically with this change though